src/pseudotransient.jl

```diff
@@ -11,6 +11,12 @@ the time-stepping and algorithm, please see the paper:
 [Coffey, Todd S. and Kelley, C. T. and Keyes, David E. (2003), Pseudotransient Continuation and Differential-Algebraic Equations,
 SIAM Journal on Scientific Computing,25, 553-569.](https://doi.org/10.1137/S106482750241044X)
 
+RobustPseudoTransient(; concrete_jac = nothing, linsolve = nothing,
+precs = DEFAULT_PRECS, alpha_initial = 1e-6,update_alpha=robust_update_alpha, adkwargs...)
```

Comment:
Put this as the documentation for RobustPT and not in PT. Cross reference from here as `See also .....`